geanypy/src/geanypy-editor.c

```diff
@@ -62,7 +62,7 @@ Editor_get_property(Editor *self, const gchar *prop_name)
 		PyObject *py_doc;
 		py_doc = (PyObject *) Document_create_new_from_geany_document(
 									self->editor->document);
-		if (py_doc && py_doc != Py_None)
+		if (!py_doc || py_doc == Py_None)
```

Comment:
Agreed, the previous one seems weird: returning `None` when `py_doc` is *not* `NULL` or `None`?  weird.

New condition seems more reasonable to me.  I wonder if `Py_RETURN_NONE` is different than `return Py_None`, but it doesn't change anything.